apps/emqx_machine/src/emqx_machine_terminator.erl

```diff
@@ -41,7 +41,7 @@ start_link() ->
 
 is_running() -> is_pid(whereis(?TERMINATOR)).
 
-%% @doc Send a signal to activate the terminator.
+%% @doc Call terminator to stop applications, then call init:stop() stop beam.
```

Comment:
Nit, I almost forget what is `terminator`,  let say `emqx terminator` to make less confusion.

Reply:
done, changed to `emqx_machine_terminator'